general/q3.py

```diff
@@ -0,0 +1,25 @@
+def secondSmallestInArray(array):
+    if len(array) < 2:
+        raise ValueError
```

Comment:
Description of error might be good but could also just let it fall over to python index error naturally